package.json

```diff
@@ -75,6 +75,7 @@
   },
   "peerDependencies": {
     "@edx/paragon": ">= 10.0.0 < 21.0.0",
+    "@edx/frontend-build": ">= 8.1.0",
```

Comment:
This is the version of frontend-build that enables `env.config` as a magic import.  So we didn't have a peer dependency here before, but we technically do now.  It's unlikely to bite anyone since 8.1.0 was quite a while ago

Reply:
For the record (Adam and I talked about this a bit offline) - the only way I could get `env.config` to work as an import was to use specialized webpack config (in frontend-build) to fallback to an empty import if the file couldn't be found.  Since the entire mechanism is predicated on that behavior, and it's implemented in frontend-build, it feels a bit unavoidable to add the peer dependency.  There's just not a way I've ever found to do this without that added magic (which was merged into frontend-build some time ago in https://github.com/openedx/frontend-build/pull/200)

Reply:
FWIW, we may have ended up needing to do something similar given the direction [this PR](https://github.com/openedx/frontend-platform/pull/440) is headed to be able to fallback to a local Paragon version in the externally hosted Paragon CSS urls or if a externally hosted Paragon CSS urls fails to load, we need to expose the paths to the locally installed Paragon theme via frontend-build such that frontend-platform has access to them (otherwise, each MFE would have to pass args to `initialize` which feels like could be avoided).